src/Discord.Net.Core/Entities/Permissions/ChannelPermission.cs

```diff
@@ -1,40 +1,29 @@
 ﻿namespace Discord
 {
-    public enum ChannelPermission : byte
+    public enum ChannelPermission : ulong
```

Comment:
Is `[FlagsAttribute]` what you were looking for? https://msdn.microsoft.com/en-us/library/system.flagsattribute(v=vs.110).aspx